src/utilities.ts

```diff
@@ -0,0 +1,3 @@
+export const defaultToTrue = (value: boolean | undefined = true): boolean => {
```

Comment:
i don't think its necessary to add `undefined` explicitly here? Especially since you already have a default assignment for when it is `undefined`.

Reply:
I wouldnt have, but TS made me. It makes sense though since in order to default undefined to true, the function has to accept undefined as a parameter type.  Now that I am including `null` like you suggested its type is `boolean | undefined | null `